.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,230 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'mkdocs.yml'
+      - '**.md'
+  push:
+    branches:
+      - '[0-9]+.[0-9]+.x'
+      - 'refs/pull/*'
+    paths-ignore:
+      - 'mkdocs.yml'
+      - '**.md'
+
+jobs:
+  composer-args:
+    name: Create default composer arguments
+    runs-on: ubuntu-latest
+    outputs:
+      default: --ansi --no-interaction --no-progress --prefer-dist
+      php8: --ansi --no-interaction --no-progress --prefer-dist --ignore-platform-req=php
+    steps:
+      - id: composer-args
+        run: |
+          echo "Setting outputs defining default and PHP 8 composer arguments"
+
+  lint-json:
+    name: Lint JSON files
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: json-syntax-check
+        run: |
+          echo "Linting composer.json"
+          jq empty composer.json
+          for JSON in .github/workflows/*.json;do
+            echo "Linting $JSON" ;
```

```suggestion
            echo "Linting $JSON"
```

you can omit the `;`, YAML `|` preserves the `\n`:
- https://yaml.org/spec/1.2/spec.html#id2760844